packages/prettier-plugin-java/src/printers/expressions.js

```diff
@@ -277,17 +278,25 @@ class ExpressionsPrettierVisitor {
       if (ctx.primaryPrefix[0].children.This !== undefined) {
         firstSeparator = "";
       }
+      // console.log("=====================");
```

Comment:
lets not merge commented out code 😄

Reply:
Yes it is still WIP because I am still trying to fix 2 bugs and I still need the printing.